explorer/interpreter/value.h

```diff
@@ -36,52 +46,58 @@ class AssociatedConstant;
 // details.
 class Value {
  public:
+  // An X-macro to apply a macro to each kind of Value.
```

Comment:
Would it be reasonable to structure these closer to what we've done for x-macros in the toolchain, with a .def file that's included?

Reply:
Sure, done.

Reply:
Oh ye bitter irony, as I work to remove the .def files on the toolchain side